pages/care.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/style/style.css">
+    <link rel="stylesheet" href="/style/care.css">
+    <title>Care</title>
+    <link rel="icon" href="../assets/plant-icon.png" type="image/icon">
+</head>
+<body>
+    <header>
+      <h1>Plant Party Fansite</h1>
+      <nav>
+        <ul>
+          <li>
+            <a class="menu-link" href="../index.html">Home</a>
+          </li>
+          <li>
+            <a class="menu-link" href="#">Care</a>
+          </li>
+          <li>
+            <a class="menu-link" href="facts.html">Facts</a>
+          </li>
+          <li>
+            <a class="menu-link" href="gallery.html">Gallery</a>
+          </li>
+        </ul>
+      </nav>
+  </header>
+  <main>
+    <div class="intro">
+      <div id="summary">
+          <p> This is a quick guide on how to take care of your plants!</p>
+      </div>
+      <div id="table-of-contents">
+        <h2>Plant Care</h2>
+        <ol>
+          <li><a href="#Light">Light 🌞</a></li>
+          <li><a href="#Watering">Watering 💦</a></li>
+          <li><a href="#Soil">Soil and Potting 🪴</a></li>
+          <li><a href="#Temperature">Temperature and Humidity 🌡</a></li>
+        </ol>
+        <h3>Other tips and tricks</h3>
+        <ul>
+          <li><a href="#Troubleshooting">Troubleshooting common issues</a></li>
+          <li><a href="#Green_Thumb">Taking your Green Thumb up a notch!</a></li>
+        </ul>
+      </div>
+    </div>
+    
+    <div class="content">
+
+      <div id="Light">
```

Comment:
The sections with id "Light", "Watering", "Soil", "Temperature", "Troubleshooting", and "Green Thumb" feel like distinct sections of a page. Prefer to replace all these non-semantic `div`s with `section` tags